contracts/token/ERC721/ERC721SingleRentalAgreement.sol

```diff
@@ -169,4 +169,39 @@ contract ERC721SingleRentalAgreement is Context, IERC721RentalAgreement, ERC165
     function supportsInterface(bytes4 interfaceId) public view override(ERC165, IERC165) returns (bool) {
         return interfaceId == type(IERC721RentalAgreement).interfaceId || super.supportsInterface(interfaceId);
     }
+
+    // ===== Getter functions ===== //
+    // For cost optimization, these functions could be removed and storage variables can be stored such
+    // that they are easily accessible from storage reads.
+    function getRentalDuration() public view returns (uint32) {
```

Comment:
I'm sorry but these function have no purpose. What do they bring over the autogenerated ones from Solidity?
If you want to keep those functions, then maybe make the related variables private, but I would prefer just not adding those.

Reply:
Forgot public variables were provided getter functions in the ABI haha, I removed it